build-image.sh

```diff
@@ -25,31 +24,30 @@ image_tag="${DOCKER_IMAGE_TAG:-$graalvm_version-java$java_major_version}" # e.g.
 #################################################
 case $graalvm_version in
    dev)   graalvm_version=$(curl -fsSL -H "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/graalvm/graalvm-ce-dev-builds/releases/latest | grep "tag_name" | cut -d'"' -f4)
-          graalvm_url="https://github.com/graalvm/graalvm-ce-dev-builds/releases/download/${graalvm_version}/graalvm-community-java${java_major_version}-linux-amd64-dev.tar.gz"
+          graalvm_url="https://github.com/graalvm/graalvm-ce-dev-builds/releases/download/${graalvm_version}/graalvm-community-java${java_major_version}-linux-ARCH-dev.tar.gz"
```

Comment:
Can we use something like `@ARCH@` or `{{ARCH}}` so that it is clearer that this is actually a placeholder that is later resolved